test/test-buildSrcSet.js

```diff
@@ -387,6 +387,21 @@ describe('SrcSet Builder:', function describeSuite() {
                         assert(src.includes(`q=${QUALITY_OVERRIDE}`));
                     }
                 });
+
+                it('should not modify input params and should be idempotent', function testSpec() {
```

Comment:
Could we add the same test for `.buildSrc` too?

Reply:
Yessir!